Makefile

```diff
@@ -1,6 +1,7 @@
-
 test:
-	@node_modules/.bin/tape test/*.js
+	@node_modules/.bin/tape test/test.js
 
 .PHONY: test
```

Comment:
the `bench` target needs to be phony too, so this line should be `.PHONY: test bench`. in a makefile you mark a target as phony when it doesn't depend on file changes to be executed, like a c recompile, but rather is executed when a developer sees fit